packages/fixtures/fixtures.coffee

```diff
@@ -15,8 +15,23 @@ do ->
         email: attributes.email
         password: attributes.password
 
+    'createTestUserWithProfile': (attributes, profileAttributes) ->
+      UserProfiles.remove({})
+      Meteor.users.remove({})
```

Comment:
I think it might be better to have a separate step that resets everything at the beginning of a test - you might want to call multiple methods if you needed a few fixtures users.